parts/dcosmastervars.t

```diff
@@ -4,6 +4,7 @@
     "apiVersionDefault": "2016-03-30",
     "apiVersionLinkDefault": "2015-01-01",
     "singleQuote": "'",
+    "escapedSingleQuote": "''",
```

Comment:
Because there are lots of different ways to express escaping depending on the parser, could you rename this to `twoSingleQuotes` or something literal like that?

Reply:
Sure